### ISSUE TITLE
Token Validation?

### ISSUE THREAD

**Issue by wenzowski** [KnowledgeGarden/BacksideServletKS#13]
We've got `IAuthMicroformat.VALIDATE` to check if a handle is registered and `IAuthMicroformat.EXISTS_EMAIL` to check if an email address is registered, but no verb yet to check if an auth token is valid.

Since [`TicketPojo`](https://github.com/OpenSherlock/TQElasticKnowledgeSystem/blob/master/src/main/java/org/topicquests/ks/TicketPojo.java) appears to create ephemeral auth tokens, wouldn't all sessions be lost if the server rebooted?

This relates to #11 in that we could solve this by using immutable [UUID](https://tools.ietf.org/html/rfc4122)'s as tokens instead, and rely on secret key signing via [JSON Web Token](https://tools.ietf.org/html/rfc7519) to make the immutable token verifiable and expirable instead.

**Comment by wenzowski** [comment]
Reading `BaseHandler` it does seem `CredentialCache` is memory-only.

**Comment by KnowledgeGarden** [comment]
Validating auth token is part of BaseHandler: if the token doesn't return
an ITicket, it's not valid. That's a space in which there is room for
serious evolution. We do not have a mechanism for invalidating a credential
unless and only unless a user logs off properly. Some websites will time
you out; we don't do that. Yes, credential cache is memory-only; it could
just as easily be an app in the new ElasticTupleSpace platform.

**Comment by wenzowski** [comment]
Yes, and that's working correctly. This issue is intended to be about a dedicated validation verb that has no side effects: all it would do is return a boolean for the validity of the auth token.
